images/router/haproxy/conf/haproxy-config.template

```diff
@@ -346,10 +346,6 @@ backend be_secure:{{$cfgIdx}}
   timeout server  {{$value}}
     {{- end }} {{/* end balance algorithm setting. */}}
```

Comment:
Is the comment `{{/* end balance algorithm setting. */}}` actually correct?

I think I'd just lose it.